### PR TITLE
[review] windows: add a resource file for parameter.dll

### PR DESCRIPTION
**Pull request by dawagner**
This file is understood by Visual Studio and embeds various read-only
properties on the generated binary such as version number, copyright
information, ...
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/01org/parameter-framework/pull/343%23issuecomment-173523759%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/343%23issuecomment-173620958%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/343%23issuecomment-173638624%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/343%23discussion_r50401625%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/343%23discussion_r50403558%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/343%23issuecomment-173523759%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%40krocard%20%40tcahuzax%20Please%20review%22%2C%20%22created_at%22%3A%20%222016-01-21T10%3A07%3A39Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dawagner%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A-1%3A%22%2C%20%22created_at%22%3A%20%222016-01-21T16%3A14%3A54Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/11178583%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tcahuzax%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%20%22%2C%20%22created_at%22%3A%20%222016-01-21T17%3A07%3A41Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%200ded38f99c4bfc5b923a6aab0e34dddb59412e03%20support/windows/Resource.rc.in%2042%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/343%23discussion_r50401625%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Their%20is%20no%20VS_FF_RELEASE%20%3F%22%2C%20%22created_at%22%3A%20%222016-01-21T13%3A45%3A44Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22Nope%2C%20cf.%20https%3A//msdn.microsoft.com/en-us/library/windows/desktop/aa381058%2528v%3Dvs.85%2529.aspx%22%2C%20%22created_at%22%3A%20%222016-01-21T14%3A03%3A29Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dawagner%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20support/windows/Resource.rc.in%3AL1-76%22%7D%7D%2C%20%22approved%22%3A%20%7B%22https%3A//github.com/krocard%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/krocard'></a>

- [ ] <a href='#crh-comment-Pull 0ded38f99c4bfc5b923a6aab0e34dddb59412e03 support/windows/Resource.rc.in 42'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/343#discussion_r50401625'>File: support/windows/Resource.rc.in:L1-76</a></b>
- <a href='https://github.com/krocard'></a> Their is no VS_FF_RELEASE ?
- <a href='https://github.com/dawagner'></a> Nope, cf. https://msdn.microsoft.com/en-us/library/windows/desktop/aa381058%28v=vs.85%29.aspx
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/343#issuecomment-173523759'>General Comment</a></b>
- <a href='https://github.com/dawagner'></a> @krocard @tcahuzax Please review
- <a href='https://github.com/tcahuzax'></a> :-1:

<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/343?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/343?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/343?approve=0'></a>&nbsp;<a href='https://github.com/01org/parameter-framework/pull/343'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by dawagner** [comment]
@krocard @tcahuzax Please review

File: support/windows/Resource.rc.in [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+// See https://msdn.microsoft.com/en-us/library/windows/desktop/aa381058%28v=vs.85%29.aspx
+VS_VERSION_INFO VERSIONINFO
+    FILEVERSION @WINRC_MAJOR@,@WINRC_MINOR@,@WINRC_PATCH@,@WINRC_TWEAK@
+    PRODUCTVERSION @WINRC_MAJOR@,@WINRC_MINOR@,@WINRC_PATCH@,@WINRC_TWEAK@
+    FILEFLAGSMASK VS_FFI_FILEFLAGSMASK
+#ifdef _DEBUG
+    FILEFLAGS VS_FF_DEBUG
+#else
+    FILEFLAGS 0x0L
```
Comment (krocard) [inline support/windows/Resource.rc.in]: Their is no VS_FF_RELEASE ?

Comment (dawagner) [inline support/windows/Resource.rc.in]: Nope, cf. https://msdn.microsoft.com/en-us/library/windows/desktop/aa381058%28v=vs.85%29.aspx

**Comment by tcahuzax** [comment]
:-1:

**Comment by krocard** [comment]
:shipit:

**Pull request merged**